assets/model_monitoring/amlcompute_components/data_drift/data_drift_compute_metrics_command/spec.yaml

```diff
@@ -1,25 +1,21 @@
 $schema: http://azureml/sdk-2-0/SparkComponent.json
-type: spark
+type: command
 
 name: data_drift_compute_metrics_command
 display_name: Data Drift - Compute Metrics
 description: Compute data drift metrics given a baseline and a deployment's model data input.
-version: 0.0.1
+version: 0.0.2
 is_deterministic: true
 
 code: ../../src
-entry:
-  file: ./data_drift_compute_metrics/run.py
+environment: azureml:model_monitor_spark_standalone:1
```

Comment:
TODO: Will update this again once the environments are available in azureml registry.